### PR TITLE
Bump djl version to 0.10.0

### PR DESCRIPTION
**Pull request by aksrajvanshi**
Bump version of demos to djl bom version - 0.10.0 and pytorch native version to 1.7.1

### CODE REVIEW

**Comment by aksrajvanshi** [comment]
@aws-samples/djl-committers

**Comment by lanking520** [comment]
Have you tested and verify everything is working?

File: spark/image-classification/build.sbt [hunk 1]
```diff
@@ -8,10 +8,10 @@ scalacOptions += "-target:jvm-1.8"
 resolvers += Resolver.jcenterRepo
 
 libraryDependencies += "org.apache.spark" %% "spark-core" % "2.3.0"
-libraryDependencies += "ai.djl" % "api" % "0.9.0"
+libraryDependencies += "ai.djl" % "api" % "0.10.0"
 
 // libraryDependencies += "ai.djl.mxnet" % "mxnet-model-zoo" % "0.9.0"
 // libraryDependencies += "ai.djl.mxnet" % "mxnet-native-auto" % "1.7.0-backport"
 
 libraryDependencies += "ai.djl.pytorch" % "pytorch-model-zoo" % "0.9.0"
```
Comment (frankfliu) [inline spark/image-classification/build.sbt:16]: update to 0.10.0

File: spark/image-classification/build.sbt [hunk 2]
```diff
@@ -8,10 +8,10 @@ scalacOptions += "-target:jvm-1.8"
 resolvers += Resolver.jcenterRepo
 
 libraryDependencies += "org.apache.spark" %% "spark-core" % "2.3.0"
-libraryDependencies += "ai.djl" % "api" % "0.9.0"
+libraryDependencies += "ai.djl" % "api" % "0.10.0"
 
 // libraryDependencies += "ai.djl.mxnet" % "mxnet-model-zoo" % "0.9.0"
```
Comment (frankfliu) [inline spark/image-classification/build.sbt:13]: this line as well

File: spark/notebook/Image_Classification_Spark.ipynb [hunk 3]
```diff
@@ -26,9 +26,9 @@
    "source": [
     "import $ivy.`org.apache.spark::spark-sql:3.0.1`\n",
     "import $ivy.`org.apache.spark::spark-mllib:3.0.1`\n",
-    "import $ivy.`ai.djl:api:0.9.0`\n",
+    "import $ivy.`ai.djl:api:0.10.0`\n",
     "import $ivy.`ai.djl.pytorch:pytorch-model-zoo:0.9.0`\n",
```
Comment (frankfliu) [inline spark/notebook/Image_Classification_Spark.ipynb:30]: 0.10.0

File: spark/notebook/Image_Classification_Spark.ipynb [hunk 4]
```diff
@@ -26,9 +26,9 @@
...
-    "import $ivy.`ai.djl.pytorch:pytorch-native-auto:1.7.0`\n",
+    "import $ivy.`ai.djl.pytorch:pytorch-native-auto:1.7.1`\n",
     "// import $ivy.`ai.djl.mxnet:mxnet-model-zoo:0.9.0`\n",
```
Comment (frankfliu) [inline spark/notebook/Image_Classification_Spark.ipynb:32]: Update the comments as well

File: spark/spark3.0/image-classification/build.sbt [hunk 5]
```diff
@@ -9,13 +9,13 @@ resolvers += Resolver.jcenterRepo
 
 libraryDependencies += "org.apache.spark" %% "spark-sql" % "3.0.1"
 libraryDependencies += "org.apache.spark" %% "spark-mllib" % "3.0.1"
-libraryDependencies += "ai.djl" % "api" % "0.9.0"
+libraryDependencies += "ai.djl" % "api" % "0.10.0"
 
 // libraryDependencies += "ai.djl.mxnet" % "mxnet-model-zoo" % "0.9.0"
 // libraryDependencies += "ai.djl.mxnet" % "mxnet-native-auto" % "1.7.0-backport"
 
 libraryDependencies += "ai.djl.pytorch" % "pytorch-model-zoo" % "0.9.0"
```
Comment (frankfliu) [inline spark/spark3.0/image-classification/build.sbt:17]: same here

File: web-demo/interactive-console/src/main/java/ai/djl/examples/jshell/ShellSpawner.java [hunk 6]
```diff
@@ -59,7 +59,7 @@ private static void extractJars(Path dir) {
                 Arrays.asList(
                         "pytorch-engine-0.9.0.jar",
```
Comment (frankfliu) [inline web-demo/interactive-console/src/main/java/ai/djl/examples/jshell/ShellSpawner.java:60]: search for all 0.9.0

**Comment by frankfliu** [comment]
> Have you tested and verify everything is working?

There should be some incompatible between 0.9.0 and 0.10.0,
At least, we should see if the code can compile

**Comment by aksrajvanshi** [comment]
yeah sorry my bad, I assumed we might have some tests running if I created a PR. But I will run all the modules and check.

**Comment by aksrajvanshi** [comment]
> > Have you tested and verify everything is working?
> 
> There should be some incompatible between 0.9.0 and 0.10.0,
> At least, we should see if the code can compile

I have tested the modules. I'm not sure whether we should update djl-ui-demo too. That project looked like it hasn't updated in a while.

**Review by frankfliu — APPROVED** [review]

**Pull request merged**